planet4-gutenberg-blocks.php

```diff
@@ -410,3 +410,109 @@ function ( $std_class_object ) {
 };
 $remove_rtl_fix();
 add_action( 'wpml_after_startup', $remove_rtl_fix, 10, 0 );
+
+$breakpoints = [
+	[
+		'screen' => 1600,
+		'width'  => '1320px',
+	],
+	[
+		'screen' => 1200,
+		'width'  => '1140px',
+	],
+	[
+		'screen' => 992,
+		'width'  => '960px',
+	],
+	[
+		'screen' => 768,
+		'width'  => '720px',
+	],
+	[
+		'screen' => 601,
+		'width'  => '540px',
+	],
+	[
+		'screen' => 577,
+		'width'  => '540px',
+	],
+];
+
+add_filter(
+	'render_block',
+	function ( $block_content, $block, WP_Block $instance ) use ( $breakpoints ) {
+		// Query param toggle to test difference.
+		if ( isset( $_GET['nofix'] ) ) {
```

Comment:
I'll remove this once review is passed. That's also the only failing test in PHP tests.